tutorials/misc/plot_ecog.py

```diff
@@ -42,13 +60,41 @@
 
 info = mne.create_info(ch_names, 1000., 'ecog').set_montage(montage)
 
+###############################################################################
+# Now that we have our electrode positions in MRI coordinates, we can load in
+# our corresponding time-series data. We then compute a time-frequency
+# representation of the data (i.e. 1-30, or 30-90 Hz).
+
+# first we'll load in the sample dataset
+raw = mne.io.read_raw_edf(misc_path + '/ecog/sample_ecog.edf')
+
+# drop bad channels
+raw.info['bads'].extend([ch for ch in raw.ch_names if ch not in ch_names])
+
+# attach montage
+raw.set_montage(montage, on_missing='ignore')
+
+# create a 1 Epoch data structure
+epoch = mne.EpochsArray(raw.get_data()[np.newaxis, ...],
+                        info=raw.info)
+
+# perform gamma band frequency averaged over entire time period
+tfr_pwr, _ = tfr_morlet(epoch, freqs=np.linspace(30, 90, 60),
+                        n_cycles=2)
+gamma_activity = tfr_pwr.data.mean(axis=(1, 2))
+
+# perform low frequency activity averaged over entire time period
+tfr_pwr, _ = tfr_morlet(epoch, freqs=np.linspace(8, 12, 4),
+                        n_cycles=2)
+low_activity = tfr_pwr.data.mean(axis=(1, 2))
```

Comment:
I would find it a lot simpler to do:

`gamma_activity  = np.sum(raw.copy().filter(30, 90).get_data() ** 2, axis=1)`
`low_activity  = np.sum(raw.copy().filter(8, 12).get_data() ** 2, axis=1)`

https://20077-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/misc/plot_ecog.html#sphx-glr-auto-tutorials-misc-plot-ecog-py